server/tests/test_route_blog.py

```
@@ -0,0 +1,34 @@
+import pytest
+from models.blog import Blog
+from models.user import User  # make sure to import your User model
+
+
+@pytest.fixture
+def test_blogs(session):
+    # Create a test admin user
+    admin = User(username="admin", email="admin@example.com", password="hashedpassword")
```

Ruff (0.13.1)

Possible hardcoded password assigned to argument: "password"

(S106)